src/core-recipe.jl

```diff
@@ -3,13 +3,13 @@
         colormap = Reverse(:RdBu),
         colorrange = Makie.automatic,
         sensors = true,
-        interpolation = ClaughTochter(),
+        interpolation = CloughTocher(),
         extrapolation = GeomExtrapolation(),
         bounding_geometry = Circle,
         enlarge = 1.2,
         markersize = 5,
         pad_value = 0.0,
-        resolution = (512, 512),
+        interp_resolution = (512, 512),
```

Comment:
this is a breaking change.

Reply:
it looks like one, but actually `resolution` was never accessible because the recipe throws an error (as resolution is a kwargs from Figure) - thus I'd classify it as bugfix instead of breaking